spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -31,27 +31,26 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
 
       omniboxController.suggestions = 100;
       expect(omniboxController.hasSuggestions()).toBe(false);
-    });
 
-    it('should return false for empty Arrays and Objects', () => {
-      omniboxController.suggestions = [];
+      omniboxController.suggestions = {test: 'me'};
       expect(omniboxController.hasSuggestions()).toBe(false);
```

Comment:
Hmm, my initial inclination is no. A function that starts with `is` or `has` should return a `Boolean` imo, not error out.